NanoCORE/DiJetSelections.cc

```diff
@@ -1,16 +1,25 @@
 #include "DiJetSelections.h"
-#include "Config.h"
-#include "DiPhotonSelections.h"
```

Comment:
`DiPhotonSelections.h` is included in `DiJetSelections.h` and the latter is included here. As a result, the former is also indirectly included here. That's the proper way to do it, so that each file is included only once.